pom.xml

```diff
@@ -703,6 +738,12 @@
         <artifactId>slf4j-nop</artifactId>
         <version>${slf4j.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.glassfish.jersey.test-framework.providers</groupId>
+        <artifactId>jersey-test-framework-provider-grizzly2</artifactId>
+        <version>${jersey.version}</version>
+        <scope>test</scope>
```

Comment:
We typically don't set the scope in the parent POM. We manage the dependency versions in the parent POM, and choose the scope in the module where it is used.